ui/src/translations/en.json

```diff
@@ -73,7 +73,7 @@
   "create_new_volume": "Create a New Volume",
   "no_volume_found": "No volumes on this node",
   "no_searched_volume_found": "No volumes matching the search",
-  "type": "Type",
+  "volume_type": "Volume Type",
   "volume_size": "Volume Capacity",
   "device_path": "Device path",
```

Comment:
There appears to be a casing difference in here, e.g. between `Volume Type` (`T`) and `Device path` (`p`). Is that intentional? Any 'rule'?